### PR TITLE
feat: release extras along with the agc package

### PR DESCRIPTION
**Pull request by evanyui**
Issue #, if available:

**Description of Changes**

[//]: #  (A description of the change that you made and the new user experience that it creates)
- Release extras into dist folder
- Audited policies and updated to include some required permissions for the newly added cdk bootstrap step during account activate, (probably some others at other step, but most significant amount of changes come from addition of cdk bootstrap)

**Description of how you validated changes**

[//]: #  (A description of you validated your changes and any relevant logs that show your change works)
- Ran package release and verify existence of extras in `/dist/`
- Ran integration tests with this version of release (including the `extras` folder) to verify the added permissions work with account activate, context deploy and running workflow.

**Checklist**

- [x] If this change would make any existing documentation invalid, I have included those updates within this PR
- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [x] I have linted my code before raising the PR
- [x] Title of this Pull Request follows Conventional Commits standard: https://www.conventionalcommits.org/en/v1.0.0/

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Review by a-li — APPROVED** [review]

File: extras/agc-minimal-permissions/lib/policy-statements.ts [hunk 1]
```diff
@@ -28,27 +28,11 @@ export class AgcPermissions {
...
-                    "CreateVpcEndpoint",
-                    "DeleteVpcEndpoints",
-                    "AllocateAddress",
-                    "AssociateAddress",
-                    "ReleaseAddress",
+                    "Create*",
+                    "Delete*",
+                    "Authorize*",
+                    "Revoke*",
+                    "*Address",
```
Comment (evanyui) [inline extras/agc-minimal-permissions/lib/policy-statements.ts:35]: Policy api threw an error as we are exceeding the limit of characters in policy document and was recommended to use wildcards as I am adding more permissions here for the purpose of (cdk bootstrap).
* Same applies to other section of the changes

Comment (elliot-smith) [reply extras/agc-minimal-permissions/lib/policy-statements.ts:35]: This breaks the whole idea of "Minimal Permissions". We can always split up the policies into two or more policies.

Comment (evanyui) [reply extras/agc-minimal-permissions/lib/policy-statements.ts]: Done

File: extras/agc-minimal-permissions/lib/permissions-stack.ts [hunk 2]
```diff
@@ -13,15 +14,18 @@ export class AgcPermissionsStack extends cdk.Stack {
       description: "managed policy for amazon genomics cli admins"
     })
 
-    let agcUserPolicy = new ManagedPolicy(this, 'agc-user-policy', {
-      description: "managed policy for amazon genomics cli users"
+    let agcUserPolicyPart1 = new ManagedPolicy(this, 'agc-user-policy-part1', {
```
Comment (elliot-smith) [inline extras/agc-minimal-permissions/lib/permissions-stack.ts:17]: Can you please change these to a reasonable name about what has been grouped as customers will see this?

Comment (evanyui) [reply extras/agc-minimal-permissions/lib/permissions-stack.ts]: Agree, done!

**Review by a-li — APPROVED** [review]

**Review by elliot-smith — APPROVED** [review]

**Pull request merged**